sig-list.md

```diff
@@ -9,7 +9,7 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 # SIGs and Working Groups
 
 Most community activity is organized into Special Interest Groups (SIGs),
-time bounded Working Groups, and the [community meeting](communication.md#weekly-meeting).
+time bounded Working Groups, and the [community meeting](communication/README.md#weekly-meeting).
```

Comment:
@dholbach Good catch!

This is generated from https://github.com/kubernetes/community/blob/2f87761d8aa7043946a10ef7123ce1d78a4096cd/generator/list.tmpl#L5. Could you make the change here?

Reply:
Sure.

Reply:
And then run `make` for generation :)